documentation/asciidoc/topics/proc_configuring_sites_manually.adoc

```diff
@@ -0,0 +1,106 @@
+[id='configuring_sites-{context}']
+= Specifying {brandname} Cluster Hostnames for Cross-Site Replication
+Specify hostnames and ports for {brandname} clusters so they can establish connections and form cross-site views.
+
+.Prerequisites
+
+* Ensure that you have the correct hostnames and port numbers for each {brandname} cluster that you plan to configure as a backup location.
+
+.Procedure
+
+. Create an **Infinispan** CR for each {brandname} cluster.
+. Specify the name of the local site with `spec.service.sites.local.name`.
+. Provide the name, URL, and port for each {brandname} cluster that acts as a backup location with `spec.service.sites.locations`.
++
+The following are example **Infinispan** CR definitions for **LON** and **NYC**:
++
+* **LON**
++
+[source,yaml,options="nowrap",subs=attributes+]
+----
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  name: {example_crd_name}
+spec:
+  replicas: 3
+  service:
+    type: DataGrid
+    sites:
+      local:
+        name: LON
+        expose:
+          type: LoadBalancer
+      locations:
+        - name: LON
+          host: infinispan-lon.myhost.com
+          port: 7900
+        - name: NYC
+          host: infinispan-nyc.myhost.com
+          port: 7900
+----
++
+* **NYC**
++
+[source,yaml,options="nowrap",subs=attributes+]
+----
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  name: {example_crd_name}
+spec:
+  replicas: 2
+  service:
+    type: DataGrid
+    sites:
+      local:
+        name: NYC
+        expose:
+          type: LoadBalancer
+      locations:
+        - name: NYC
+          url: infinispan-nyc.myhost.com
```

Comment:
```suggestion
          host: infinispan-nyc.myhost.com
```

Reply:
@dmvolod Can you please explain this a little more? I don't understand what you mean. How is the host generated on the primary site?

Reply:
@oraNod in case of static config, this is LoadBalancer host name as describer in #648

Reply:
I think I see that it relates to https://github.com/infinispan/infinispan-operator/issues/648 

When `expose.type: LoadBalancer` then the host is generated automatically at the local site. That enhancement allows the automatically generated host to get populated in the Infinispan CR so users don't need to do that.

Reply:
@dmvolod gotcha. #648 needs merge first though, right? at the moment, you still need to enter the host for the local site.

Reply:
@oraNod  in this case users will need to get `LoadBalancer` host name from config (infinispan.yaml inside ConfigMap) or service and define it on the second site manually

Reply:
@dmvolod yeah, getting the host name is part of the manual process. I could make that a little more explicit in the docs but I kind of feel like it might add complexity because it might not be a LoadBalancer in the case of, like, minishift. 

At the moment I've got this as a prereq in the manual xsite procedure: 

"Ensure that you have the correct hostnames and port numbers for each {brandname} cluster that you plan to configure as a backup location."

I feel like that covers it and users should be savvy enough to find those details.

Reply:
@dmvolod When will there be a PR to fix #648 ? If it's going to be a while I can add a comment to that issue so we remember to update the docs.

Reply:
@oraNod I don't know right now. I can share my OCP cluster locally with you via e-mail to be able look how to use xsite on OCP AWS cluster.